static/style.css

```diff
@@ -0,0 +1 @@
enu__logout,.top-menu__profile,.top-menu__settings{text-align:left;white-space:nowrap}.top-menu__user{position:relative;white-space:nowrap;text-align:end}.top-menu__user>a{padding-right:0}.top-menu__user,.top-menu__write{flex-basis:48%}.top-menu__a11y{position:absolute;top:-20em;right:0;left:0;text-align:center}.top-menu__a11y ul{margin:0 auto;display:block}.top-menu__a11y a:focus{margin-top:20em}.top-menu__logout button{font:inherit}.tab{border-radius:1px 1px 0 0;border-bottom:3px solid transparent;height:48px;width:8rem}.tab:hover{color:#070809;border-color:#5c7d07}.tab:active,.tab:focus{color:#fdfdfc;background:#7cad00}.tab.active{border-color:#7cad00}ul.top-menu__user-submenu{display:none;position:absolute;right:0;top:2rem;background:#fdfdfc}ul.top-menu__user-submenu li{display:block;margin:0;padding:0}ul.top-menu__user-submenu a,ul.top-menu__user-submenu button{padding:.5rem 1rem;width:100%}ul.top-menu__user-submenu a:hover,ul.top-menu__user-submenu button:hover{background:#42413d;color:#fdfdfc}.top-menu__user:focus .top-menu__user-submenu,.top-menu__user:focus-within .top-menu__user-submenu,.top-menu__user:hover ul,ul.top-menu__user-submenu:hover{display:block}.icon{width:20px;height:20px;display:inline-block;margin-right:.5em;stroke:currentcolor;stroke-width:4px;fill-opacity:0}a:hover .icon,button:hover .icon{fill-opacity:1}.gear-inner-circle,.globe,.house-outline,.magnifier-frame,.magnifier-handle,.no-fill,.rocket-body,.rocket-fin{fill:none}.globe-node{stroke:none}.house-outline{fill:#7cad00}.globe-node,.house2 .house-outline{fill:#008fb3}.house3 .house-outline{fill:#74255a}.mask-outline{fill:#b30077}.gear-outer-circle{fill:#070809}.exit-door,.gear-inner-circle{fill:#f8f8f6}.exit-arrow,.rocket-fin{fill:#d12005}.house-window,.rocket-body{fill:#fdfdfc}#fave,#post,.rocket-flame{fill:#ffc31f}.gear-tooth,.mask-features{fill:#42413d}#x{stroke:#fdfdfc;stroke-width:1px}.h-card .p-name{font-size:2rem}.h-card .fq-username{font-family:"Noto Mono","Droid Sans Mono",Consolas,monospace}input[type=checkbox].collapse--toggle{display:none}.collapse--lbl-toggle{padding:0 .5em}.collapse--content{display:none}.collapse--toggle:checked+.collapse--lbl-toggle+.collapse--content{display:block}@media screen and (min-width:1600px){html{font-size:20px}}@media screen and (min-width:800px){.small-screen-only{display:none;visibility:hidden}}@media screen and (max-width:799px){html{font-size:16px}.main-content{margin-top:0;flex-direction:column}.write-status{padding-top:6rem}.status,.write-status-form{max-width:100%}.logo{height:32px}.top-menu{height:42px}.top-menu a{padding:.5rem 0;overflow:hidden;white-space:nowrap;max-width:20px;height:42px}.logo a,ul.top-menu__user-submenu a,ul.top-menu__user-submenu button{max-width:8em}.anon-user-controls{margin:5rem auto;text-align:center}.page-end,.page-header,main>section{padding:0 1rem}}@media screen and (max-width:480px){html{font-size:14px}.write-status{padding-top:3rem}.logo{display:none}.top-menu,.top-menu a{height:40px}}
```

Comment:
is this supposed to be checked in? @y6nH

Reply:
Yeah, I was in two minds about that. The idea is that somebody wanting to work on the back-end only won't have to concern themself with compiling front-end stuff, especially since I've gone for Gulp rather than the Sass binary as before. Many people will already have Node / NPM / Gulp already set up, but it lowers the barrier to entry for those who don't. Since these are small, cross-platform files, I canme down on that side. Maybe rearranging the folder structure would make it clearer.

Reply:
Ugh.... trying to figure out how to do this. I like the idea of Gulp, but I'm not 100% sure I want to pull in Node, but I _also_ really do not like the idea of trying to keep in-tree artifacts up to date.

Reply:
I haven't looked into it, but there must be a bot that can run Gulp on push, so that at least we know it matches the source each time.

Reply:
[Looks as if it's pretty simple with Travis.](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-gulp)

Reply:
OK, Travis is now running Gulp and passing.

Reply:
@y6nH awesome! that won't get back into the repo if it's mismatched, though, and doesn't check that it's up-to-date afaict?

Reply:
I've added a Travis deployment config that ought to push back to GitHub, but haven't seen it in action yet.